src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Categories/Blocks/showcase_card.html.twig

```diff
@@ -23,16 +23,26 @@
  * International Registered Trademark & Property of PrestaShop SA
  *#}
 
-<div class="helper-card card">
-  <div class="helper-card__left helper-card__customer shape-one">
-    <img src="{{ asset('themes/new-theme/scss/img/helper-card/customer@3x.png') }}" class="img-fluid">
+{% if not isShowcaseCardClosed %}
+  <div id="categoriesShowcaseCard" class="helper-card card">
+    <div class="helper-card__left helper-card__category shape-one">
+      <img src="{{ asset('themes/new-theme/scss/img/helper-card/category@3x.png') }}" class="img-fluid img-rtl">
+    </div>
+    <div class="helper-card__right">
+      <h2>{{ 'Set categories'|trans({}, 'Admin.Catalog.Feature') }}</h2>
+      <p>{{ 'Categories are essential, as they enable you to group equivalent products. This helps customers find their way through the variety of your catalog.'|trans({}, 'Admin.Catalog.Feature') }}</p>
```

Comment:
From my side, I have the following wording, _Create a menu tree to group your products and lead commercial operations so your customers can easily find what they are looking for. You won’t find such a time saver!_, I don't know why it is not the same. @TristanLDD?

Reply:
Warning: this page has both an Empty State and a Helper Card 😉

The Empty state appears when the listing is empty while the Helper Card appears all the time, until the user closes it (oh, I see there's going to be an issue here, when listing is empty do we display both ?).

This is the PR for the Helper Card whose text comes from https://projects.invisionapp.com/share/FQNYNAE6RDJ#/screens

Reply:
It's a bit confusing, there are mockups in Invision & Zeplin and sometimes they differ. :/ In addition, I couldn't find Empty state for Categories, @matks do you know where I can get it from?

Reply:
ping @LouiseBonnard @TristanLDD for working, which one should be used?